src/containers/scenes/aoi-scene/index.js

```diff
@@ -223,7 +223,8 @@ function AOIScene(props) {
     setBasemap({
```

Comment:
we should set the basemap only once on the map initialization and then use the setBasemap only on the scene component